DEPERECATED.md

```diff
@@ -0,0 +1,3 @@
+# DEPRECATED starting from 1.3.0
```

Comment:
Maybe put a top level description of what this file is for and what rules are (feature is removed starting from the specified version, etc) 

And then having a list of features that will be removed in a specific version.

Description

Version 1.4.0:
* item 1
* item 2

Version 1.5.0:
* ..

etc